### DISCUSSION TITLE
Calendar and Contacts

### DISCUSSION THREAD

**Discussion by voodooca** [iluvadev/XstReader#20] [General]
Hello, I found this program after looking for other OST - PST converters. It opens the OST file and I can see everything - very nice. Am I able to get CSV exports from this for either the contacts or calendar items? I wasnt able to see anything that covered this idea in the readme. Thanks!

**Comment by voodooca** [comment]
Sorry, I have an update - I was using the original developers version and it was opening my Ost file fine.
When I tried grabbing your more updated version, it seems to stall/freeze (not responding).
The ost file is large, so I am wondering if that is a limitation also.
Thanks

**Comment by ShivamRathore99** [comment]
I am facing a problem converting an OLM file to a PDF, please suggest to me the best software to convert an OLM file to a PDF file without any problem.

**Comment by kenthomson001** [comment]
How can I convert a large OST file to PST without losing emails, attachments, or folder structure?
